apps/console/src/features/connections/components/create/enterprise-connection-create-wizard.tsx

```diff
@@ -261,7 +262,7 @@ export const EnterpriseConnectionCreateWizard: FC<EnterpriseConnectionCreateWiza
      * @param value - URL to check.
      */
     const isLoopBackCall = (value: string) => {
-        return (!(URLUtils.isLoopBackCall(value))) ?
+        return (!(URLUtils.isLoopBackCall(value) && commonConfig?.blockLoopBackCalls)) ? 
             undefined : t("console:develop.features.idp.forms.common." +
                 "internetResolvableErrorMessage");
```

Comment:
Instead of using a ternary and negative conditions, keeping conditions positive and using good ol' if statements would improve the readability drastically here. WDYT 😊 ?

```suggestion
        if (commonConfig?.blockLoopBackCalls && URLUtils.isLoopBackCall(value)) {
           return "t("console:develop.features.idp.forms.common." +
                "internetResolvableErrorMessage")"
        } 
        
        return undefined;
```

Let's also update the function name too to something like `checkValueIsLoopBackCall` (or more nice name) if possible.

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/4557/commits/65bc53d15d9bc8aff7a1880b4ead8f3bf7aa6723